lms/djangoapps/ci_program/migrations/0006_auto_20221222_2000.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.14 on 2022-12-22 20:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ci_program', '0005_program_status'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='program',
+            old_name='sample_content_for',
+            new_name='sample_content',
+        ),
+        migrations.RemoveField(
+            model_name='program',
+            name='support_program_for',
+        ),
+        migrations.AddField(
```

Comment:
Tis best to perform additive changes only to databases. Window the change out for the next release, then remove the older field when appropriate. The Syllabus app for example uses this field, and will have to be changed.

Reply:
Hmmmm I see your point, I'm afraid I wasn't thinking that far ahead. 
We might need a short session first thing tomorrow morning together with @nakitamccool before we actually commit to deploying it.

Reply:
@bravoalpha79 Happy to catch up in the morning on this.